src/build/build.ts

```diff
@@ -59,10 +62,35 @@ export async function build(
   logger.info(`Deleting build/ directory...`);
   await del([buildDirectory]);
 
+  let babelConfigJson = null;
+  if (options.compile) {
+    try {
+      const babelConfigPath = path.resolve(config.root, '.babelrc');
```

Comment:
let's not read the babelrc. Please use the hard-coded list of presets here: https://github.com/PolymerLabs/polyserve/blob/master/src/compile-middleware.ts#L24